android/app/src/main/java/io/mosip/registration_client/api_services/DashBoardDetailsApi.java

```diff
@@ -28,17 +28,15 @@
 public class DashBoardDetailsApi implements DashBoardPigeon.DashBoardApi {
     private UserDetailDao userDetailDao;
     private RegistrationRepository registrationRepository;
-    private PreRegistrationDataSyncService preRegistrationData;
     public static final String PREFERRED_USERNAME = "preferred_username";
     SharedPreferences sharedPreferences;
     private Context context;
 
     @Inject
-    public DashBoardDetailsApi(Context context, UserDetailDao userDetailDao, RegistrationRepository registrationRepository,PreRegistrationDataSyncService preRegistrationData){
```

Comment:
this seems to be removing prereg data sync service, this feature is needed

Reply:
@jainhitesh9998 - I confirmed from @SachinPremkumar that these changes were done for developer testing purpose only and not required in actual workflow. Hence removed